tests/System.Text.Primitives.Tests/Encoding/Utf8EncoderTests.cs

```diff
@@ -4,79 +4,6 @@ namespace System.Text.Utf8.Tests
 {
     public class Utf8EncoderTests
```

Comment:
Could you please add some tests testing the scenario of partial encoding/decoding, i.e. give the method too small output buffer, and when it return false, slice the input buffer, give it a new output buffer, and then call the method again to keep encoding/decoding.